airbyte-integrations/connectors/destination-teradata/src/main/resources/spec.json

```diff
@@ -21,28 +21,153 @@
         "title": "User",
         "description": "Username to use to access the database.",
         "type": "string",
-        "order": 4
+        "order": 1
       },
       "password": {
         "title": "Password",
         "description": "Password associated with the username.",
         "type": "string",
         "airbyte_secret": true,
-        "order": 5
+        "order": 2
       },
       "schema": {
         "title": "Default Schema",
         "description": "The default schema tables are written to if the source does not specify a namespace. The usual value for this field is \"public\".",
         "type": "string",
-        "examples": ["public"],
-        "default": "public",
+        "examples": ["airbyte_td"],
```

Comment:
this change seems unnecessary

Reply:
public is keyword in Teradata system, and it will not allow to create schema with public as default value.  Changed it to airbyte_td